Add morpheus project resume (#134)
volkamerlab/volkamerlab_org: content/projects/MORPHEUS.md

---
title: Morpheus
date: 2023-08-25
publishdate: 2023-08-25
weight: 20
nav: false
people:
- key: floriane.odje
- key: andrea.volkamer

> funding:

Reviewer: also add the collaborators category and add the people from BfR and FMP, including links.

- name: Bundesministerium für Bildung und Forschung (BMBF)
  link: https://www.bmbf.de/bmbf/en/ministry/research-funding/research-funding_node.html
---

> The MORPHology-based Endocrine Disruptor Screening [Morpheus] project aims to develop deep learning models to predict the effects of substances and identify characteristic fingerprints based on morphological and molecular input data.

Reviewer: ```suggestion
The computational part of the MORPHology-based Endocrine DisrUptor Screening (_Morpheus_) project aims to develop deep learning models to predict the effects of substances and identify characteristic fingerprints based on morphological and molecular input data.
```

<!--more-->

> Morpheus project aims to enable the efficient identification and characterisation of endocrine-like substances with harmful effects on humans and the environment. Those substances are often studied in time-consuming experiments involving animals raising ethical issues. Our partner (BfR and FMP) propose to study them using in vitro methods such as cell painting assay.

Reviewer: ```suggestion
The BMBF-funded Morpheus project aims to enable the efficient identification and characterization of endocrine-like substances with harmful effects on humans and the environment. Those substances are often studied in time-consuming experiments involving animals raising ethical issues. Our partner (BfR and FMP) and us (UdS) propose to study them using in vitro methods such as cell painting assay combined with _in silico_ modelling.
```

> Cell painting assay output, called morphological fingerprints, reveals how the substances impact cell viability. This information is easily interpretable by computers, as fingerprints, thus we propose to develop machine learning and deep learning methods that leverage information from them. Such tools would help in predicting substance effects, identifying characteristic fingerprints of hazardous compounds and predicting morphological fingerprints using molecules as input.

Reviewer: ```suggestion
Cell painting assay data reveals morphological changes in the cell, and thus, how the substances impact cell viability.  Such data can nowadays be produced with high throughput (e.g. [CIL](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5721342/)). This information, after being converted to numbers as _morphological fingerprints_, is easily interpretable by computers. Thus, we propose to develop machine learning and deep learning methods that leverage information from morphological fingerprints and combine them with structural information, e.g. structural fingerprints. Such tools will help to predict substance effects and to identify characteristic fingerprints of hazardous compounds.
```

> These models will be iteratively trained and validated based on freely available data, as well as applied and optimized on new profiles successively generated by the project partners from BfR and FMP. Finally, the generated in silico models will be applied to effectively search extensive substance libraries and investigate different family compounds, which will also be experimentally verified.

Reviewer: ```suggestion
These models will be iteratively trained and validated based on freely available data (CIL), as well as applied and optimized on new profiles successively generated by our project partners from BfR and FMP. 

Finally, the generated in silico models will be applied to effectively search large substance libraries, which will be experimentally verified.
```
